superset-frontend/src/components/IconButton/index.tsx

```diff
@@ -41,6 +41,15 @@ const StyledImage = styled.div`
   height: ${({ theme }) => theme.gridUnit * 18}px;
   margin: ${({ theme }) => theme.gridUnit * 3}px 0;
 
+  .default-db-icon {
+    font-size: ${({ theme }) => theme.gridUnit * 9}px;
```

Comment:
Font sizes shouldn't be set using grid units. There is `theme.typography.sizes` for that.

Reply:
Looks like we don't have a typography size large enough for what you're going for here though. Doing math with the typography sizes sounds pretty unsavory. If we don't want to add an `xxxl` size maybe grid units are actually the right fit here.

Reply:
Oh yah forgot about that. Maybe i'll just put down the actual font-size then.